common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -80,33 +73,35 @@ const Tab = styled(Space).attrs({
 const TabPanel = styled(Space)`
   background: ${(props) => props.theme.color('cream')};
 `;
+
 type Props = {
-  worksRouteProps: WorksRouteProps;
-  imagesRouteProps: ImagesRouteProps;
-  workTypeAggregations: CatalogueAggregationBucket[];
+  query: string;
+  sort: string | undefined;
```

Comment:
Is there a reason these can't be `sort?: string`?

Reply:
Nope, that's been changed.